client/wfnews-war/src/main/angular/src/app/app.routing.ts

```diff
@@ -35,4 +35,4 @@ const PANEL_ROUTES: Routes = [
 ];
 
 
-export const ROUTING = RouterModule.forRoot(PANEL_ROUTES, { relativeLinkResolution: 'legacy' });
+export const ROUTING = RouterModule.forRoot(PANEL_ROUTES, { relativeLinkResolution: 'legacy', useHash:true });
```

Comment:
We shouldn't need the `useHash` here. We have routing handled in Tomcat so the workaround isn't needed on our dev deploy